added more requirements (#17)
peppelinux/draft-demarco-oauth-nonce-endpoint: draft-demarco-nonce-endpoint.md

The **Nonce** MUST satisfy the following requirements:

- It MUST be opaque to receiving Clients.

> - It MUST be unguessable, by cryptographic randomness or encryption.

Reviewer: if we can avoid the "or" here, lots of implementation complexity will go away.

Author: see https://github.com/peppelinux/draft-demarco-oauth-nonce-endpoint/pull/17/files#r1481755422

> - It MUST be unguessable, by cryptographic randomness or encryption.

Reviewer: ```suggestion
- It MUST be encrypted.
```

Author: https://github.com/peppelinux/draft-demarco-oauth-nonce-endpoint/pull/17/files#r1481755422

> - It MUST be unguessable, by cryptographic randomness or encryption.

Author: ```suggestion
- It MUST be made unguessable through the use of cryptographic randomness.
```

Reviewer: I'll take this, but I still think we could be clearer, this is very open ended... which is undesirable IMO.

> - It MUST be used once and for a short time.

Reviewer: ```suggestion
- It MUST have a short validity period
```

Author: since both short validity and usable once are requirements I propose to slit that sentence in two distinct entries, see my proposal below

Author: https://github.com/peppelinux/draft-demarco-oauth-nonce-endpoint/pull/17/files#r1481757786

> - It MUST be used once and for a short time.

Author: ```suggestion
- It MUST be used once.
- It MUST have a short validity period.
```
